src/pages/index.tsx

```
@@ -85,6 +85,42 @@ const Square: React.FC = () => {
+                page: 0,
+              });
+              setTags([]);
+              history.pushState(
+                { domain: CATEGORY_TEAM_EN[i] },
+                CATEGORY_TEAM_EN[i],
+                `${CATEGORY_TEAM_EN[i]}`,
+              );
+              useDocTitle('都是自己人啦~');
+            }}
+            className="wrapper"
+            aria-hidden="true"
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)